src/auth/auth.ts

```diff
@@ -346,6 +346,10 @@ export const isAuthSettled = (state: AuthState) => {
   );
 };
 
+export const hasAcceptedTermsOfService = (state: AuthState) => {
```

Comment:
It's always nice to have the signature of a function documented in types.

```suggestion
export const hasAcceptedTermsOfService = (state: AuthState): boolean => {
```